.travis.yml

```diff
@@ -1,5 +1,9 @@
+dist: bionic
 sudo: false
 language: ruby
 rvm:
-  - 2.3.1
-before_install: gem install bundler -v 1.12.5
+- 2.5.7
+- 2.6.5
+cache:
+  bundler: true
+before_install: gem install bundler -v 2.1.3
```

Comment:
Is this line even necessary? Do we do this elsewhere and I just didn't notice?

Reply:
Yea, I tried that [here](https://travis-ci.org/ManageIQ/query_relation/jobs/632311667) but it failed the build